smart-contracts/ProdEnvRunbook.md

```diff
@@ -0,0 +1,13 @@
+# Production Environment Setups For Scripts
+
+The following document will describe how to set up your environment in the `smart-contracts` folder to be able to talk to betanet effectively and run operational or maintenance actions.
+
+1). Ensure that the `ETHEREUM_PRIVATE_KEY` variable in your .env file is set to the correct role for the action that you want to do. For example, if you want to pause the bridgebank, ensure that the private key in your env file has the pauser role capability.
```

Comment:
fine for now, though would be nice in future if we can use specific variables for each key rather than one private key, eg: OPERATOR_PRIVATE_KEY, PAUSER_PRIVATE_KEY, etc.

Reply:
The scripts are only designed to be run with one role at once. A single script should not be sending transactions from multiple different addresses.